### ISSUE TITLE
getDayClasses function on Ipad

### ISSUE THREAD

**Issue by niconorsync** [arca-computing/MultipleDatePicker#107]
Hi! I have seriuos problems to perform the library on my ipad. I think the reason is the function getDayClasses(day) that calls every second and slows down all the calendar functionality in the ipad making it imposible to use. However the same page in my laptop runs perfectly.
Personally I'd choose ng-class to manage the days css classes instead of:
div class="text-center picker-day {{getDayClasses(day)}}"

I hope it'd would help you in some way.

**Comment by mgohin** [comment]
Hi,

thanks for your feedback.
Could you try to modify the lib on your side to check if it's better ?

I don't have any iPad or 'slow' device to test :/

**Comment by niconorsync** [comment]
Hi mgohin! I've just tested it in an ipad and the performance is better using ng-class. However the conflicts selecting/unselecting days persists.
I finally solved it by deleting this class: `.picker-day:not(.picker-off):not(.picker-empty):hover{background-color:#C6000B;color:#fff;cursor:pointer}`
Now doesnt have any style on hover event but it fixed the selection problems. 
Thanks for your response anyway! and sorry for my argentinian english..
